docs/resources/service-providers.md

```diff
@@ -4,79 +4,194 @@ order: 5
+  version             Print the application binary version information
+
+Flags:
+  -h, --help                help for gaiad
+      --home string         directory for config and data (default "/Users/tobias/.gaia")
+      --log_format string   The logging format (json|plain) (default "plain")
+      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
+      --trace               print out full stack trace on errors
+
+Use "gaiad [command] --help" for more information about a command.
+```
+
+For each displayed command, you can use the `--help` flag to get further information. 
+
+```bash
+gaiad query --help
+Usage:
+  gaiad query [flags]
+  gaiad query [command]
+
+Aliases:
+  query, q
+
+Available Commands:
+  account                  Query for account by address
+  auth                     Querying commands for the auth module
+  bank                     Querying commands for the bank module
+  block                    Get verified data for a the block at given height
+  distribution             Querying commands for the distribution module
+  evidence                 Query for evidence by hash or for all (paginated) submitted evidence
+  gov                      Querying commands for the governance module
+  ibc                      Querying commands for the IBC module
+  ibc-transfer             IBC fungible token transfer query subcommands
+  mint                     Querying commands for the minting module
+  params                   Querying commands for the params module
+  slashing                 Querying commands for the slashing module
+  staking                  Querying commands for the staking module
+  tendermint-validator-set Get the full tendermint validator set at given height
+  tx                       Query for a transaction by hash in a committed block
+  txs                      Query for paginated transactions that match a set of events
+  upgrade                  Querying commands for the upgrade module
+
+Flags:
+      --chain-id string   The network chain ID
+  -h, --help              help for query
+
+Global Flags:
+      --home string         directory for config and data (default "/Users/tobias/.gaia")
+      --log_format string   The logging format (json|plain) (default "plain")
+      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
+      --trace               print out full stack trace on errors
+
+Use "gaiad query [command] --help" for more information about a command.
 ```
 
-It allows you to set a default value for each given flag. 
+### Remote Access to gaiad
+
+When choosing to remote access a Full Node and gaiad, you need a Full Node running and gaia installed on your local machine.
+
+`gaiad` is the tool that enables you to interact with the node that runs on the Cosmos Hub network, whether you run it yourself or not.
```

Comment:
This line could be moved earlier.

Reply:
how far earlier do you have in mind? Beginning of this chapter or even at the start of the whole article?